src/ReaderView/ReaderView.cpp

```diff
@@ -0,0 +1,40 @@
+#include "ReaderView/ReaderView.h"
+
```

Comment:
Shouldn't we include `QPushButton`, `QHBoxLayout`, `QLabel`, `QVBoxLayout` here?

Reply:
`#include <QtWidgets>` comes from `#include "ReaderView/ReaderView.h"`. I will add that to the `ReaderView.cpp` as well just in case.

Reply:
Oh, missed that it's `<QtWidgets>` instead of `<QWidget>`. Apparently  `<QtWidgets>` contains a lot of stuff. 
So the better way to do is to include `<QWidget>` in header file as we only need `QWidget` there, and in cpp we can include `<QtWidgets>` or just we are using.